genie-docs/src/docs/asciidoc/_metrics.adoc

```diff
@@ -301,7 +301,7 @@ It will have to be added if one is desired otherwise metrics are just published
 |status, exceptionClass, scriptUri
 
 |genie.services.agentConfiguration.reloadProperties.timer
-|Time taken to refresh the set of properties forwarded to bootstrapping agents
+|Time taken to retrieve the set of properties forwarded to bootstrapping agents
```

Comment:
nit: These property change fixes aren't really related to the map change commit

Reply:
This documents the slight change of semantic for this metric, I think it's in the right commit.